src/iuse_actor.cpp

```diff
@@ -3769,8 +3769,8 @@ bool place_trap_actor::is_allowed( Character &p, const tripoint &pos,
     const trap &existing_trap = here.tr_at( pos );
     if( !existing_trap.is_null() ) {
         if( existing_trap.can_see( pos, p ) ) {
-            p.add_msg_if_player( m_info, _( "You can't place a %s there.  It contains a trap already." ),
-                                 name );
+            p.add_msg_if_player( m_info, _( "You can't place a %s there.  It contains %s already." ),
+                                 name, existing_trap.is_benign() ? _( "a deployed object" ) : _( "a trap" ) );
```

Comment:
```suggestion
            p.add_msg_if_player( m_info,
                                 existing_trap.is_benign()
                                     ? _( "You can't place a %s there.  It contains a deployed object already." )
                                     : _( "You can't place a %s there.  It contains a trap already." ),
                                 name );
```
Avoids conflicts with possible identical strings in other places and also gives translators more context.

Reply:
Good idea, done.